.github/workflows/release.yml

```diff
@@ -69,35 +63,59 @@ jobs:
           aws s3 sync ./build "$BUCKET_PATH"
         working-directory: ./docs
 
-      - name: Push kolena, kolena-client dists to Test PyPI
+      - name: Push 'kolena' dist to Test PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           repository-url: https://test.pypi.org/legacy/
 
-      - name: Push kolena, kolena-client dists to PyPI
+      - name: Push 'kolena' dist to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
 
-      - name: Install twine for package distribution on CodeArtifact
-        run: pip3 install twine
+      - name: Create GitHub release with 'kolena' build as artifact
```

Comment:
Nit: do we need this given current release flow? Every release is published to pypi, and currently no non-release build.

Reply:
We don't really need the artifact, but I figure it's good to include anyway. We definitely want to keep this action in general so that automated releases with changelogs are created when tags are pushed.

Reply:
Ended up removing the artifact in #32. It isn't necessary when the client is shipped via PyPI